app/jobs/update_model_job.rb

```diff
@@ -6,9 +6,25 @@
 class UpdateModelJob < ApplicationJob
   queue_as :low
 
-  def perform(klass, where_vals, new_attrs)
+  def perform(klass, where_vals, new_attrs, user_id=0)
+    if user_id == 0
+      UpdateModelJob.notify_exception(ArgumentError, klass, where_vals, new_attrs)
+    elsif !User.exists?(user_id)
```

Comment:
Stylistic, but I'd recommend:

```rb
    elsif (user = User.find_by(id: user_id)).nil?
```

And then I'd use this `user` later in:

```rb
      Audited.audit_class.as_user(user) do
```

That way we avoid two different calls to find the user (I haven't used `User.exists?` before like this)